packages/transport/src/api/usb.ts

```diff
@@ -354,21 +354,34 @@ export class UsbApi extends AbstractApi {
     }
 
     public async closeDevice(path: string) {
-        const device = this.findDevice(path);
+        let device = await this.findDevice(path, true);
```

Comment:
this is the important change that makes it work on my machine. adding `true` here. Other places could maybe force enumeration too but I wanted to make changes minimal.